youtube/youtube-extension/src/Home.js

```diff
@@ -0,0 +1,189 @@
+/* eslint-disable react/jsx-no-target-blank */
+import React from "react";
+import "./styles/style.css";
+import Dragula from "react-dragula";
+import Youtube from "./helper/youtube";
+import { WindowOpener } from "./components/window-opener";
+import ContentstackUIExtension from "@contentstack/ui-extensions-sdk";
+export class Home extends React.Component {
```

Comment:
Change Class components to Functional components because that is easy to use, more readable, have less line of code and comes in best practices